deploy/sre-build-test/50-build-test.CronJob.yaml

```diff
@@ -79,6 +86,12 @@ spec:
                     ;;
                   Complete)
                     echo "$(date): Build Complete"
+                    JOBS_TO_DELETE=$(oc -n "${POD_NS}" get job --no-headers || true | \
+                                                              grep -v "${JOB_NAME}" | \
+                                                              grep -e "${JOB_PREFIX}" -e '0/1' | \
```

Comment:
Intent based on slack conversation is to clear the alert.  To do that we need to delete only the failed jobs.  This line will match any job that has failed (assuming "0/1" is a failure) AND any job with the prefix.  I.e. it will delete all the jobs regardless of status.

Recommend this instead:

```bash
JOBS_TO_DELETE=$(oc -n "${POD_NS}" get jobs -o json | jq -r ".items[] | select(.metadata.name | contains(\"$JOB_PREFIX\")) | select(.status.succeeded != 1) | .metadata.name")
```

Reply:
Hmm. If we have `jq` available..

Since we only care about failed jobs and the job this is running in is _not_ failed, could also do:
```bash
JOBS_TO_DELETE$(oc -n "${POD_NS}" get jobs -o jsonpath='{.items[?(@.status.failed)].metadata.name}')
```

Reply:
I can add this, it sounds fair, also tested and the latter command works (no `jq` on this container)

Reply:
did it @jewzaam, please review to your satisfaction